NaviGator/gnc/navigator_thrust_mapper/navigator_thrust_mapper/thruster_map.py

```diff
@@ -172,14 +172,16 @@ def from_urdf(
                             transmission.name
                         )
                     )
+
                 t_ratio = transmission.actuators[0].mechanicalReduction
-                if ratio != -1 and ratio != t_ratio:
+
+                if t_ratio < 0:
                     raise Exception(
-                        "Transmission {} has a different reduction ratio (not supported)".format(
-                            t_ratio
-                        )
+                        "Thruster mapper does not allow negative mechanical reduction."
```

Comment:
`f"Thruster mapper does not allow negative mechanical reduction (transmission {i}: {t_ratio})."` ?